doc/admin-guide/plugins/rate_limit.en.rst

```diff
@@ -161,10 +161,10 @@ The following options are available:
 .. option:: --iprep_buckets
 
    The number of LRU buckets to use for the IP reputation. A good number here
-   is 10, but can be configured. The reason for the different buckets is to
-   account for a pseudo-sorted list of IPs on the frequency seen. Too few buckets
-   will not be enough to keep such a sorting, rendering the algorithm useless. To
-   function in our setup, the number of buckets must be less than ``100``.
+   is `10`, which is the default, but can be configured. The reason for the different
```

Comment:
Need two ticks around 10. Otherwise this is italicized instead of a code space string.

There's a few examples of this in this patch, but just marking this one.